### ISSUE TITLE
err 520 POST message

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#526]
when posting a new message (in an existing discussion) I got a 520 err:

```
POST http://localhost:4000/api/v1/messages
{"subject":"test 1","body":"salut moi-même","identities":[{"identifier":"dev@caliopen.local","type":"email"}],"discussion_id":"117bb4c1-a4a5-4076-8796-93498de1834a","parent_id":"83ce01be-21de-4e69-8bb2-a947710bcfa3"}
```

```
{
	"errors": [{
		"code": 520,
		"description": "",
		"component": "collection_message",
		"values": 520,
		"message": "_prepare_search() takes exactly 1 argument (3 given)",
		"property": "MergePatchError",
		"type": "Unknown Error"
	}]
}
```

**Comment by iamdey** [comment]
I think, it happened when I POST a new message with a parent_id.

When I create a new message (in an existing discussion) without parent_id and then when I try to PATCH, it fails too:

```
PATCH http://localhost:4000/api/v1/messages/19d46690-ca4b-436e-a975-5f55f409757d
{"current_state":{"parent_id":""},"parent_id":"be22827e-b1c1-4fe7-a5f5-4855b3e80991"}
```
```
{
	"errors": [{
		"code": 422,
		"description": "PATCH payload was malformed or unprocessable",
		"component": "message",
		"values": 422,
		"message": "document is not valid, can't insert it into db: <{\"body_plain\": \"Rogue field\", \"body_html\": \"Rogue field\"}>",
		"property": "MergePatchError",
		"type": "Patch Unprocessable"
	}]
}
```

...
Hmm whitout parent_id it fails anyway :s

**Comment by sapiens-sapide** [comment]
Fix with commit #9922984
